tests/error/misc_errors/result_tag_too_big.py

```diff
@@ -1,11 +1,8 @@
-from guppylang.std.builtins import result, py
+from guppylang.std.builtins import result, comptime
 from tests.util import compile_guppy
 
-# TODO use this once https://github.com/CQCL/guppylang/issues/599 lands
-# from guppylang.prelude._internal.checker import TAG_MAX_LEN
-# BIG_TAG = "a" * (TAG_MAX_LEN + 1)
+TAG_MAX_LEN = 200
 
 @compile_guppy
-def foo(y: bool) -> None:
-    # each tick or cross is 3 bytes. The cross sends the tag over the limit.
-    result("✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅❌", y)
+def foo(y: bool) -> None:    
+    result(comptime("a" * (TAG_MAX_LEN + 1)), y)
```

Comment:
I think we shouldn't change this since it's testing that we consider the utf8 encoding of the tag instead of its Python `len`